specs/linux_file_system_hierarchy.md

```diff
@@ -176,6 +176,18 @@ It is recommended not to place binaries in this directory
 that are not useful for invocation from a shell (such as daemon binaries);
 these should be placed in a subdirectory of `/usr/lib/` instead.
 
+### `/usr/include/`
+
+Programmatic APIs header files, typically used when compiling `C/C++` programs.
+Packages may place such files either directly in `/usr/include/`,
+or in a subdirectory.
+The subdirectory may be named after the package or project providing it.
+Packages may place architecture-dependent header files and directories in a
+`/usr/include/arch-id/` subdirectory,
+following the identifiers defined on the
+[Multiarch Architecture Specifiers (Tuples)](https://wiki.debian.org/Multiarch/Tuples)
+list.
+
```

Comment:
I had the following additions in my version:
```
The corresponding library shall be placed in `$libdir`, see below.
```
and
```
Using a subdirectory is recommended if the package installs multiple files.
It is also necessary if multiple versions of the same headers shall be coinstallable.
```
ad
```
Note that to use the headers, flags for the compiler might be required
to bring the appropriate subdirectory into the search path.
See man pages
[`pc(7)`](https://www.mankier.com/5/pc),
[`pkgconf(1)`](https://www.mankier.com/1/pkgconf),
[gcc's Search Path](https://gcc.gnu.org/onlinedocs/cpp/Search-Path.html),
and other compiler specific documentation.
```

The mention of `/usr/include` also needs to be dropped from the intro where we say we don't document it.

Reply:
added